tests/unit/fem/test_transfer.cpp

```diff
@@ -75,555 +75,482 @@ void vectorcoeff(const Vector& x, Vector& y)
-                     GridFunction X(c_fespace);
-                     GridFunction X_cmp(c_fespace);
-                     GridFunction Y_exact(f_fespace);
-                     GridFunction Y_std(f_fespace);
-                     GridFunction Y_test(f_fespace);
-                     coeff_order = 1;
-                     if (vectorspace == 0)
-                     {
-                        FunctionCoefficient funcCoeff(&coeff);
-                        X.ProjectCoefficient(funcCoeff);
-                        Y_exact.ProjectCoefficient(funcCoeff);
-                     }
-                     else
-                     {
-                        VectorFunctionCoefficient funcCoeff(dimension, &vectorcoeff);
-                        X.ProjectCoefficient(funcCoeff);
-                        Y_exact.ProjectCoefficient(funcCoeff);
-                     }
-
-                     Y_std = 0.0;
-                     Y_test = 0.0;
-
-                     referenceOperator->Mult(X, Y_std);
-
-                     Y_std -= Y_exact;
-                     REQUIRE(Y_std.Norml2() < 1e-12 * Y_exact.Norml2());
-
-                     testTransferOperator.Mult(X, Y_test);
-
-                     Y_test -= Y_exact;
-                     REQUIRE(Y_test.Norml2() < 1e-12 * Y_exact.Norml2());
-
-                     referenceOperator->MultTranspose(Y_exact, X);
-                     testTransferOperator.MultTranspose(Y_exact, X_cmp);
-
-                     X -= X_cmp;
-                     REQUIRE(X.Norml2() < 1e-12 * X_cmp.Norml2());
-
-                     delete referenceOperator;
-                     delete f_fespace;
-                     delete c_fespace;
-                     if (geometric == 0)
-                     {
-                        delete f_fec;
-                     }
-                     delete c_fec;
-                  }
-               }
-            }
-         }
-      }
+      case VecSpace::H1: return "H1";
+      case VecSpace::VectorH1: return "Vector H1";
+      case VecSpace::ND: return "Nedelec";
+      case VecSpace::RT: return "Raviart-Thomas";
    }
+   return "";
 }
 
-
-TEST_CASE("variable_order_transfer")
+TEST_CASE("Transfer", "[Transfer]")
 {
-   for (int vectorspace = 0; vectorspace <= 3; ++vectorspace)
+   auto vectorspace = GENERATE(VecSpace::H1, VecSpace::VectorH1, VecSpace::ND,
+                               VecSpace::RT);
+   auto geometric = GENERATE(true, false);
+   auto simplex = GENERATE(true, false);
+   dimension = GENERATE(2, 3);
+
+   int order = 2;
+   int ne = 2;
+
+   int fineOrder = geometric ? order : 2*order;
+
+   INFO("  Vector space: " << VecSpaceName(vectorspace) << '\n' <<
+        "  Dimension:    " << dimension << '\n' <<
+        "  Simplex:      " << simplex << '\n' <<
+        "  Elements:     " << std::pow(ne, dimension) << '\n' <<
+        "  Coarse order: " << order << '\n' <<
+        "  Fine order:   " << fineOrder << '\n' <<
+        "  Geometric:    " << geometric);
```

Comment:
Nice! Do you know if there is many other places left with spurious `std::cout`?

Reply:
Yeah, unfortunately I think there are quite a few other places that still remain.